chef-powershell/tasks/gem_build.rb

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
It doesn't seem like this task is used for building the gem since you have that in a powershell script. So you can probably remove this file.

Reply:
done